js/main.js

```diff
@@ -8,7 +8,7 @@ $(document).ready(function(){
                   ['006E00', 'FFD732'], ['6E008C', 'FFD732']];
     var color = Math.floor((Math.random() * 12));
     var logoID = color + 1;
-    if (color < 10) {
+    if (color < 9) {
```

Comment:
< 9 is 1-8. < 10 is 1-9. I think we want the latter.

Reply:
@walterbender 10 is what we already have; we are getting this error: https://github.com/sugarlabs/www/issues/477; so I suspected it may be due to the `color +1`